odcockpit/include/plugins/environmentviewer/EnvironmentViewerGLWidget.h

```diff
@@ -23,6 +23,7 @@
 
 #include <map>
 #include <vector>
+#include <memory>
```

Comment:
@HangYin2015 Could you please revert the changes to this file so that I can complete the PR?